azuredeploy.json

```diff
@@ -263,71 +318,28 @@
         {
             "type": "Microsoft.Resources/deployments",
             "apiVersion": "2015-01-01",
-            "name": "appsettings",
+            "name": "initaccelerator",
             "dependsOn": [
-                "[resourceId('Microsoft.CognitiveServices/accounts', variables('cognitiveServicesAllInOneName'))]",
                 "[resourceId('Microsoft.Web/Sites/sourcecontrols', variables('functionAppName'), 'web')]"
             ],
             "properties": {
                 "mode": "Incremental",
                 "templateLink": {
-                    "uri": "[concat('https://',variables('functionAppName'),'.azurewebsites.net/api/init-kb?code=',listkeys(concat(resourceId('Microsoft.Web/sites', variables('functionAppName')), '/host/default/'),'2019-08-01').functionKeys.default)]",
+                    "uri": "[concat('https://raw.githubusercontent.com/jennifermarsman/cognitive-search-qna-solution/',variables('branch'),'/linkedazuredeploy.json')]",
```

Comment:
Use a variable for this if we want to change the path /jennifer/....

Reply:
okay